src/gentropy/l2g.py

```diff
@@ -419,16 +419,16 @@ def __init__(
         # Reading the study index
         study_index = StudyIndex.from_parquet(session, study_index_path)
 
-        # Generate evidence and save file:
-        (
-            locus_to_gene_prediction.to_disease_target_evidence(
-                credible_sets, study_index, locus_to_gene_threshold
+        # Generate evidence and save file - if there's any evidence:
```

Comment:
Related to the comment above, do you think that the lack of study metadata is a big enough reason not to return evidence?